apps/system/i2c/Makefile

```diff
@@ -125,8 +127,17 @@ install:
 
 endif
 
+ifeq ($CONFIG_BUILTIN_APPS)$(CONFIG_SYSTEM_I2CTOOL),yy)
```

Comment:
mistake fixed in:
https://github.com/Samsung/TizenRT/pull/2439#event-1959436602